constraint/pkg/client/regolib/src.go

```diff
@@ -43,11 +50,14 @@ audit[response] {
 	}
 	inventory[inv]
 	data.templates["{{.Target}}"][constraint.kind].violation[r] with input as inp with data.inventory as inv
+	spec := get_default(constraint, "spec", {})
+	enforcementaction := get_default(spec, "enforcementaction", "deny")
```

Comment:
Could we test:

1) That enforcementAction non-default gets passed through
2) that the default enforcmentaction is deny

in unit tests?

Reply:
done